packages/calcite-design-tokens/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to calcite-design-tokens
+
+You want to contribute? Nice! Below are some guidelines for ensuring that your contribution makes sense for everybody.
+
+## Reporting Issues
+
+Found a problem? Want a new feature?
+
+- See if your issue or idea has [already been reported](issues).
+- Provide detailed reproduction instructions as well as what behavior is expected.
+
+## Contributing to tokens
+
+### Read [<root>/CONTRIBUTING.md](../../CONTRIBUTING.md)
+
+These docs include initial setup, commit message format guidelines and more.
```

Comment:
What do you think about changing everything above to:

```md
# Contributing to `calcite-design-tokens`

This document contains information about contributing to the `calcite-design-tokens` package. Please read the general [CONTRIBUTING](../../CONTRIBUTING.md) document first, which includes initial setup, commit message format guidelines, and more.
```

And then re-number the headers for the rest of the file, e.g.

```md
## Project scripts

### Build
```

Info about reporting issues is already described in a lot more detail in the general CONTRIBUTING doc